### DISCUSSION TITLE
혹시 제페토 앱 최신버전에서 멀티 테스트 잘되나요?

### DISCUSSION THREAD

**Discussion by zeze103** [naverz/zepeto-studio-kor#1188] [5. Q&A]
제페토 앱 3.15.0 이상 버전에서 QR 생성하거나 테스트 링크로 게임은 실행은 에러와 함께 실행되지 않았습니다.
그래서 월드를 1.6으로 올리고 에셋 리임포트도 해서 빌드를 해도 작동하지 않았습니다.

PC에서는 아래 에러와  함께 해당 씬 안에 모든 제페토스크립트들이 NullReferenceException 에러를 발생했습니다.

NullReferenceException: Object reference not set to an instance of an object
ZEPETO.Script.Extensions.QualitySettingExtension..ctor (ZEPETO.Script.ZepetoScriptBuildManifest manifest) (at <318f9e0c271f451f917eb63f5344d284>:0)
ZEPETO.Script.Extensions.ZepetoScriptContextInitializer+<GetExtensions>d__26.MoveNext () (at <318f9e0c271f451f917eb63f5344d284>:0)
ZEPETO.Script.ZepetoScriptContext..ctor (System.Boolean debugMode) (at <318f9e0c271f451f917eb63f5344d284>:0)
ZEPETO.Script.ZepetoScriptContextComponent.Awake () (at <318f9e0c271f451f917eb63f5344d284>:0)
UnityEngine.GameObject:SetActive(Boolean)
ZEPETO.Script.ZepetoScriptContextInitializerComponent:Awake()

에러를 발생한 스크립트 이미지 입니다.

![플레이_에러](https://user-images.githubusercontent.com/31619436/185359420-b9e16609-4bf3-413b-9e30-70808e6ad2f1.PNG)

3.15 이상 앱과 PC에서만 실행되지 않지만, 제페토 앱 3.14버전이 설치된 모바일에서는 정상 작동되고 있습니다.
다른 분들도 문제 없이 잘되고  계시나요?;;

**Comment by hwisulee** [comment]
[테스트 기기]
제페토 앱 버전 3.15.1 안드로이드 : 노트 20 울트라
제페토 앱 버전 3.15.0 IOS : 아이폰 13 pro
유니티 에디터, 월드 1.6.0
정상 작동되는 것 확인했습니다

**Comment by zeze103** [comment]
프로젝트를 지우고 새로 만들어서 전부 옮겨서 해결되나 했었는데
다시 버그 발생 되었습니다.

**Reply by yoonjoohwan** [reply]
혹시 문제 해결하셨나요?

**Reply by zeze103** [reply]
예상하기에는 코드 컨버팅 중에 강제종료되면 생기는 문제 인것 같습니다만, 정확하게 원인은 못찾았습니다;;
새로 프로젝트 만들고 하고 있는데 아직까지는 버그 발생하지 않았습니다;

**Reply by YSHFM** [reply]
혹시 플라스틱scm쓰신다면,,,, 그것 때문이실 수도 있습니다. 
제페토 패키지랑 플라스틱이랑 서로 충돌나는 건지 어떤건지 모르겠지만, 저같은 경우는 여러 문제가 자주 발생했었습니다.

**Reply by zeze103** [reply]
역시 그게 문제 였군요;;;
안그래도 새로 프로젝트 만들때 플라스틱scm 연결안하고 사용하니 버그가 없었습니다;;
